addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1455,8 +1455,8 @@ options.registry.OptionsTab = options.Class.extend({
      */
     async customizeButtonStyle(previewMode, widgetValue, params) {
         await this._customizeWebsiteVariables({
-            [`btn-${params.button}-outline`]: widgetValue === 'outline',
-            [`btn-${params.button}-flat`]: widgetValue === 'flat',
+            [`btn-${params.button}-outline`]: widgetValue === 'outline' ? 'true' : 'false',
+            [`btn-${params.button}-flat`]: widgetValue === 'flat' ? 'true' : 'false',
```

Comment:
Maybe we should add the protection at a lower level? Actually I'd prefer this to be python-side and thus generic, no?

Reply:
That seems a good idea. :)